checks/cs50/2017/x/speller/checks.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Speller(Checks):
+
+    @check()
+    def exists(self):
+        """dictionary.c, dictionary.h, and Makefile exist"""
+        super(Speller, self).exists("dictionary.c")
+        super(Speller, self).exists("dictionary.h")
+        super(Speller, self).exists("Makefile")
```

Comment:
Is simpler syntax possible? Is this a Python 2/3 thing? If so, can we abstract this messiness away?